x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -123,8 +126,9 @@ const getColumnConfigs = ({
         }),
         order: 60,
         render: (index) => {
-          return Number(index.documents ?? 0).toLocaleString();
+          return <DocCountCell indexName={index.name} httpSetup={http} docCountApi={docCountApi} />;
```

Comment:
`DocCountCell`’s props are `{ indexName, docCountApi }`, but it’s being called with an extra `httpSetup={http}` prop here. That looks like a leftover and will be an unknown prop on the function component.

I think this should just be:

```suggestion
return <DocCountCell indexName={index.name} docCountApi={docCountApi} />;
```